src/test/scala/io/iohk/ethereum/vm/MockWorldState.scala

```diff
@@ -12,7 +12,8 @@ object MockWorldState {
 case class MockWorldState(
   accounts: Map[Address, Account] = Map(),
   codeRepo: Map[Address, ByteString] = Map(),
-  storages: Map[Address, MockStorage] = Map()
+  storages: Map[Address, MockStorage] = Map(),
+  headers: Map[BigInt, ByteString] = Map()
```

Comment:
to have limited number of hashes in storage, to be consistent with `state.context.env.blockHeader.number` which is the latest block and to test case when I ask for block header hash that does not exist in storage yet.
Will change 
`headers: Map[BigInt, ByteString] = Map()` => `numberOfHashes: Int`